testing/utils/cmake_test.cmake

```diff
@@ -141,6 +142,17 @@ function(add_cmake_test mode source_or_dir)
       message(FATAL_ERROR "${mode} mode not one of the valid modes (config|build|install) by add_cmake_build_test")
     endif()
 
+    if(RAPIDS_TEST_SERIAL)
+      set_tests_properties(${test_name} PROPERTIES RUN_SERIAL ON)
+      if(TEST ${test_name}_configure)
+        message(STATUS "${test_name}_configure is serial")
```

Comment:
Slightly inconsistent with a status message for the configure but not the build.

Reply:
Ohh that was a debug line, will remove it.